toml/traits.hpp

```diff
@@ -280,7 +282,7 @@ using enable_if_t = typename std::enable_if<B, T>::type;
 // ---------------------------------------------------------------------------
 // return_type_of_t
 
-#if __cplusplus >= 201703L
+#if __cplusplus >= 201703L && defined(__cpp_lib_is_invocable) && __cpp_lib_is_invocable>=201710
```

Comment:
It seems that this value should be `201703`.

cf.
- https://en.cppreference.com/w/cpp/feature_test
- https://github.com/microsoft/STL/blob/main/stl/inc/yvals_core.h#L1211


```suggestion
#if __cplusplus >= 201703L && defined(__cpp_lib_is_invocable) && __cpp_lib_is_invocable>=201703
```